frontend/test/pytest/test_measurements_shots_results.py

```diff
@@ -0,0 +1,433 @@
+
+        expected = circuit()
+        result = qjit(circuit)()
+        assert np.allclose(result, expected, atol=0.05)
+
+    def test_pauliy_expectation(self, backend):
+        """Test that PauliY expectation value is correct"""
+        n_wires = 2
+        n_shots = 5000
+        dev = qml.device(backend, wires=n_wires, shots=n_shots)
+
+        theta = 0.432
+        phi = 0.123
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.RX(theta, wires=[0])
+            qml.RX(phi, wires=[1])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliY(wires=0)), qml.expval(qml.PauliY(wires=1))
+
+        expected = circuit()
+        result = qjit(circuit)()
+        assert np.allclose(result, expected, atol=0.05)
+
+    def test_hadamard_expectation(self, backend):
+        """Test that Hadamard expectation value is correct"""
+        n_wires = 2
+        n_shots = 5000
+        dev = qml.device(backend, wires=n_wires, shots=n_shots)
+
+        theta = 0.432
+        phi = 0.123
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.RY(theta, wires=[0])
+            qml.RY(phi, wires=[1])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.Hadamard(wires=0)), qml.expval(qml.Hadamard(wires=1))
+
+        expected = circuit()
+        result = qjit(circuit)()
+        assert np.allclose(result, expected, atol=0.05)
+
+    def test_hermitian(self, backend):
+        """Test expval Hermitian observables with shots."""
+
+        @qjit
+        @qml.qnode(qml.device(backend, wires=3, shots=1000))
+        def circuit(x, y):
+            qml.RX(x, wires=0)
+            qml.RX(y, wires=1)
+            qml.CNOT(wires=[0, 1])
+            A = np.array(
+                [[complex(1.0, 0.0), complex(2.0, 0.0)], [complex(2.0, 0.0), complex(1.0, 0.0)]]
+            )
+            return qml.expval(qml.Hermitian(A, wires=2) + qml.PauliX(0) + qml.Hermitian(A, wires=1))
+
+        with pytest.raises(
+            RuntimeError,
+            match="Hermitian observables do not support shot measurement",
+        ):
+            circuit(np.pi / 4, np.pi / 4)
+
+    def test_paulix_pauliy(self, backend):
+        """Test that a tensor product involving PauliX and PauliY works correctly"""
+        n_wires = 3
+        n_shots = 10000
+        dev = qml.device(backend, wires=n_wires, shots=n_shots)
+
+        theta = 0.432
+        phi = 0.123
+        varphi = -0.543
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.RX(theta, wires=[0])
+            qml.RX(phi, wires=[1])
+            qml.RX(varphi, wires=[2])
+            qml.CNOT(wires=[0, 1])
+            qml.CNOT(wires=[1, 2])
+            return qml.expval(qml.PauliX(wires=0) @ qml.PauliY(wires=2))
+
+        expected = circuit()
+        result = qjit(circuit)()
+        assert np.allclose(result, expected, atol=0.05)
+
+    # def test_pauliz_hadamard_prod(self, backend):
```

Comment:
How come this one is commented out? If the test doesn't work, might be best to use 
```python
@pytest.mark.xfail(reason="...")
```

so we don't lose track of this test

Reply:
It's uncommented now! We got the support working for all observables (except `Hermitian`) :)

Reply:
Hey @josh146,  To get `Hermitian` support, we need get `LAPACK` support to obtain `eigenvalues` and `unitary` of a `Hermitian` matrix. As discussed with @mlxd, it will be an issue to link against `LAPACK` on `Windows` systems. So I started with Linux systems and created a PR [569](https://github.com/PennyLaneAI/pennylane-lightning/pull/569). It works fine on Perlmutter with pre-install `LAPACK` lib. We still need some tests for other Linux system. Per `Windows` support, I plan to use `ExternalProject_Add` function [here](https://cmake.org/cmake/help/latest/module/ExternalProject.html) to build `LAPACK` if `LAPACK` is not found after I'm back from vacation. Will keep you posted on the progress. BTW, this [story](https://app.shortcut.com/xanaduai/story/51606/add-hermitian-obs-support-for-measurement-with-shot-noise) was created to track the progress on the `Hermitian` support.